documentation/src/sops/ocular-physiology/oct/oct.md

```diff
@@ -189,26 +226,44 @@
         2. `[ID]_[YYYYMMDD]_[HHMMSS]_3D_MACULA(6x6GRUD).csv`
```

Comment:
Please delete `[ID]_` The CSVs contain data of all participants, thus it starts with date of saving, not with ID.